client/my-sites/stats/stats-module/index.jsx

```diff
@@ -179,6 +180,10 @@ class StatsModule extends Component {
 					</SectionHeader>
 				) }
 				<Card compact className={ cardClasses }>
+					<StatsModuleAvailabilityWarning
```

Comment:
Maybe we could check for file download stats type before rendering? It doesn't seem like we plan on using this warning elsewhere.
```suggestion
					statType === 'statsFileDownloads' && <StatsModuleAvailabilityWarning
```